aspnetcore/tutorials/first-web-api.md

```diff
@@ -151,6 +151,8 @@ The Swagger page `/swagger/index.html` is displayed. Select **GET** > **Try it o
 * The response code, body, and headers.
 * A drop down list box with media types and the example value and schema.
 
+If the Swagger page doesn't appear, see [this GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/21647).
```

Comment:
Did you mean to link to a different issue?

Reply:
No, that's the right one.

Reply:
I think it would be better to include the relevant info from that issue in this doc. Then we keep the reader in docs, where the content is localized.

Reply:
It's a pretty rare issue. We have 100K PV/month on this and only a couple folks have had that problem. I'm only guessing on what the problem is, there might be other reasons, and the issue is the right place to update those reasons. If we get more than 2 hits every quarter, we can think about moving it into the doc. Right now I don't want to pollute the doc with very rare error information.

Reply:
Our localization is no better than google translate.

Reply:
> I fixed it by going to Project Properties -> Debug and setting Hosting Model to "Out of Process".

We can't include that in the doc. Perhaps there's some reason that would fix it, but it doesn't make sense to me. Let's see how much more info we get. Folks that hit this problem can  easily provide feedback in the right place.